src/main/java/com/bathanh/exercise/basic/number/Exercise01.java

```diff
@@ -1,18 +1,23 @@
 package com.bathanh.exercise.basic.number;
 
 public class Exercise01 {
-    public int USCLN(int a, int b) {
-        while (a != b) {
-            if (a > b)
-                a = a - b;
-            else
-                b = b - a;
+
+    public int calculateGCD(final int a,final int b) {
+        int A = a;
+        int B = b;
+        while (A != B) {
+            if (A > B){
```

Comment:
Thiếu một khoảng trống sau )
```suggestion
            if (A > B) {
```

Reply:
done